src/main/resources/assets/curios/lang/ru_ru.json

```diff
@@ -1,21 +1,21 @@
 {
   "curios.name": "Curios",
   "curios.slot": "Слот",
-  "curios.cosmetic": "Косметика:",
-  "commands.curios.set.success": "Количество слотов типа «%s» было изменено на %d для %s",
-  "commands.curios.add.success": "%d слотов типа «%s» было добавлено для %s",
-  "commands.curios.remove.success": "%d слотов типа «%s» было убрано у %s",
-  "commands.curios.unlock.success": "Слот «%s» был включён для %s",
-  "commands.curios.lock.success": "Слот «%s» был отключён для %s",
-  "commands.curios.clear.success": "Слот «%s» был очищен у %s",
-  "commands.curios.clearAll.success": "Все слоты были очищены у %s",
-  "commands.curios.reset.success": "Сброшены слоты у %s",
-  "argument.curios.type.unknown": "Неизвестный тип слота: «%s»",
-  "item.curios.amulet": "Любопытный амулет",
-  "item.curios.ring": "Любопытное кольцо",
-  "item.curios.crown": "Любопытная корона",
-  "item.curios.knuckles": "Любопытный кастет",
-  "curios.identifier.curio": "Аксессуар",
+  "curios.cosmetic": "Косметический",
+  "commands.curios.set.success": "Тип %s был установлен на %d слоты у %s",
+  "commands.curios.add.success": "%d слоты типа %s были добавлены к %s",
+  "commands.curios.remove.success": "%d слотов типа %s было убрано у %s",
+  "commands.curios.unlock.success": "Слот %s был разблокирован для %s",
+  "commands.curios.lock.success": "Слот %s был заблокирован для %s",
+  "commands.curios.clear.success": "Слот %s был очищен у %s",
+  "commands.curios.clearAll.success": "Все слоты были очищены для %s",
+  "commands.curios.drop.success": "В слоте %s для %s предметы сброшены",
+  "commands.curios.dropAll.success": "Со всех слотах %s предметы сброшены",
+  "commands.curios.reset.success": "Слоты для %s сброшены",
+  "commands.curios.replace.success": "%s слот для %s заменён на %s",
+  "argument.entity.options.curios.description": "Инвентарь существа curios",
+  "argument.curios.type.unknown": "Неизвестный тип слота цурио %s",
+  "curios.identifier.curio": "Цурио",

```

Comment:
I'll need some clarity on this change. My understanding is that this translates literally to "curio" whereas the previous translation is "accessory". The correct translation is "accessory" in terms of its meaning and not the name of the mod. It is only this way in English because curio has a similar meaning, but I don't know if this is the same for Russian. If it's not, then this should not be changed.